### DISCUSSION TITLE
How can we modify the backdrop classes ?

### DISCUSSION THREAD

**Discussion by lucie1704** [themesberg/flowbite#805] [Q&A]
Hello, I'd like to change the backdrop classes but I don't wanna go doing all that js stuff of creating the modal manually, is there no way to change backdrop classes maybe just through a tag attribute like 
```
<button data-drawer-target="navigation-drawer" backdropClasses="bg-red-500/50" data-drawer-show="navigation-drawer"> Open drawer </button>
<div id="navigation-drawer"></div>
```

Thanks for the help !

**Comment by charanpasham** [comment]
I don't think there is a way to do this.
A quick hack would be to just override the default backdrop class bg-gray-900/50 with your desired style.

`.bg-gray-900\/50
{
 background-color: rgba(0, 23, 160, 0.5) !important;
}`

You can also use a variable for the color

`:root {
 --backdrop-color: 0, 23, 160;
}

.bg-gray-900\/50
{
 background-color: rgba(var(--backdrop-color, 0.5) !important;
}
`
